src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -104,34 +112,45 @@ function preprocessLayer(layer, provider) {
 }
 
 /**
- * Init the geometry layer of the Scene.
- */
-ApiGlobe.prototype.init = function init() {
-    const map = this.scene.getMap();
-    map.tiles.init(map.layersConfiguration.getGeometryLayers()[0], map.layersConfiguration.lightingLayers[0]);
-};
-
-/**
  * Add the geometry layer to the scene.
  */
-ApiGlobe.prototype.addGeometryLayer = function addGeometryLayer(layer) {
+ApiGlobe.prototype.addGeometryLayer = function addGeometryLayer(layer, parentLayerId) {
```

Comment:
It will be clearer to do 

`layerParent.addGeometryLayer(layer)` than to `api.addGeometryLayer(layer, parentLayerId)`

To deepen the issue
`layerTrees` or `sceneConfiguration`  looks like a `Map` object. 
We could imagine : 
`Map().layerParent.addGeometryLayer(layer)` or
`Map().addGeometryLayer(layer, parentLayerId)`